src/Analyzers/CSharp/Tests/UseVarTestExtensions.cs

```diff
@@ -2,189 +2,100 @@
-            { CSharpCodeStyleOptions.VarForBuiltInTypes, offWithInfo },
-        };
-
-    public static OptionsCollection PreferImplicitTypeWithInfo(this AbstractCodeActionOrUserDiagnosticTest test)
-        => new OptionsCollection(test.GetLanguage())
-        {
-            { CSharpCodeStyleOptions.VarElsewhere, onWithInfo },
-            { CSharpCodeStyleOptions.VarWhenTypeIsApparent, onWithInfo },
-            { CSharpCodeStyleOptions.VarForBuiltInTypes, onWithInfo },
-        };
-
-    public static OptionsCollection PreferExplicitTypeWithSilent(this AbstractCodeActionOrUserDiagnosticTest test)
-        => new OptionsCollection(test.GetLanguage())
-        {
-            { CSharpCodeStyleOptions.VarElsewhere, offWithSilent },
-            { CSharpCodeStyleOptions.VarWhenTypeIsApparent, offWithSilent },
-            { CSharpCodeStyleOptions.VarForBuiltInTypes, offWithSilent },
-        };
-
-    public static OptionsCollection PreferImplicitTypeWithSilent(this AbstractCodeActionOrUserDiagnosticTest test)
-        => new OptionsCollection(test.GetLanguage())
-        {
-            { CSharpCodeStyleOptions.VarElsewhere, onWithSilent },
-            { CSharpCodeStyleOptions.VarWhenTypeIsApparent, onWithSilent },
-            { CSharpCodeStyleOptions.VarForBuiltInTypes, onWithSilent },
-        };
-
-    public static OptionsCollection PreferExplicitTypeWithNone(this AbstractCodeActionOrUserDiagnosticTest test)
-        => new OptionsCollection(test.GetLanguage())
-        {
-            { CSharpCodeStyleOptions.VarElsewhere, offWithNone },
-            { CSharpCodeStyleOptions.VarWhenTypeIsApparent, offWithNone },
-            { CSharpCodeStyleOptions.VarForBuiltInTypes, offWithNone },
-        };
-
-    public static OptionsCollection PreferImplicitTypeWithNone(this AbstractCodeActionOrUserDiagnosticTest test)
-        => new OptionsCollection(test.GetLanguage())
-        {
-            { CSharpCodeStyleOptions.VarElsewhere, onWithNone },
-            { CSharpCodeStyleOptions.VarWhenTypeIsApparent, onWithNone },
-            { CSharpCodeStyleOptions.VarForBuiltInTypes, onWithNone },
-        };
-
-#endif
-
-    public static OptionsCollection PreferExplicitTypeWithError(this AbstractCodeActionOrUserDiagnosticTest_NoEditor test)
-        => new OptionsCollection(test.GetLanguage())
-        {
-                    { CSharpCodeStyleOptions.VarElsewhere, offWithError },
-                    { CSharpCodeStyleOptions.VarWhenTypeIsApparent, offWithError },
-                    { CSharpCodeStyleOptions.VarForBuiltInTypes, offWithError },
-        };
-
-    public static OptionsCollection PreferImplicitTypeWithError(this AbstractCodeActionOrUserDiagnosticTest_NoEditor test)
-        => new OptionsCollection(test.GetLanguage())
+    public static OptionsCollection PreferImplicitTypeWithError(this AbstractCodeActionOrUserDiagnosticTest_NoEditor<TestHostDocument, TestHostProject, TestHostSolution, TestWorkspace> test)
```

Comment:
Maybe add `using` for `AbstractCodeActionOrUserDiagnosticTest_NoEditor<TestHostDocument, TestHostProject, TestHostSolution, TestWorkspace>` ?